config.example.py

```diff
@@ -6,4 +6,5 @@
 # @author : Raphael Cerveaux (sundowndev)
 
 google_api_key=''
-google_cx_id=''
\ No newline at end of file
+google_cx_id=''
+firefox_exe_path = r'C:\Program Files\Mozilla Firefox\firefox.exe'
```

Comment:
This is not working. You setup a executable path for your own installation of Firefox. This  will not work for Linux users.